stock_picking_show_backorder/README.rst

```diff
@@ -0,0 +1,62 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
```

Comment:
Provide link as in the template: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst